src/migrations/2020_03_11_000003_add_st_louis_taxes.php

```diff
@@ -0,0 +1,47 @@
+        ]);
+
+        $st_louis_earnings_tax = DB::table('taxes')->insertGetId([
+            'name' => 'St. Louis Earnings Tax',
+            'class' => StLouis::class,
+        ]);
+
+        $st_louis_employer_tax = DB::table('taxes')->insertGetId([
+            'name' => 'St. Louis Payroll Expense Tax',
+            'class' => StLouisEmployer::class,
+        ]);
+
+        DB::table($this->tax_areas)->insert([[
+            'tax_id' => $st_louis_earnings_tax,
+            'home_governmental_unit_area_id' => $st_louis_gua_id,
+            'work_governmental_unit_area_id' => $st_louis_gua_id,
+            'based' => TaxArea::BASED_ON_WORK_LOCATION,
+        ]]);
+
+        DB::table($this->tax_areas)->insert([[
+            'tax_id' => $st_louis_employer_tax,
+            'work_governmental_unit_area_id' => $st_louis_gua_id,
+            'based' => TaxArea::BASED_ON_WORK_LOCATION,
+        ]]);
```

Comment:
`A St Louis Earnings tax must be withheld from all wages of residents of the city, and from wages earned within the city from non-residents.`

shouldn't these be based on either?

Reply:
Ac 1 is for the earnings tax and yes it should be weither

Ac 4 is for the employers tax and it is "4. EMPLOYER liability for "Payroll Expense Tax" should be created on all wages earned in St. Louis. " so I think the second one is right

Reply:
ya, I didn't pay enough attention :p

Reply:
All good, thanks you for reviewing and checking, I had to double check to make sure.